thrift/history.thrift

```diff
@@ -292,6 +292,7 @@ struct RecordChildExecutionCompletedRequest {
   30: optional i64 (js.type = "Long") initiatedId
   40: optional shared.WorkflowExecution completedExecution
   50: optional shared.HistoryEvent completionEvent
+  60: optional i64 (js.type = "Long") startedId
```

Comment:
nit: just out of curiosity. What is the difference between startedId and iniatedId?

Reply:
They are the eventIDs of childworkflowinitated event and childworkflowstarted event